jwst/flatfield/flat_field.py

```diff
@@ -202,14 +208,14 @@ def apply_flat_field(science, flat, inverse=False):
     # Update the variances using BASELINE algorithm.  For guider data, it has
     # not gone through ramp fitting so there is no Poisson noise or readnoise
     if not isinstance(science, datamodels.GuiderCalModel):
-        flat_data_squared = flat_data**2
+        flat_data_squared = flat_data ** 2
         science.var_poisson /= flat_data_squared
         science.var_rnoise /= flat_data_squared
-        science.var_flat = science.data**2 / flat_data_squared * flat_err**2
+        science.var_flat = science.data ** 2 / flat_data_squared * flat_err ** 2
```

Comment:
Curse flake8. To me the original syntax (without whitespace) is more natural and readable (to a human).

Reply:
Agreed 😞